charts/timescale-config/templates/custom-secret-scripts.yaml

```diff
@@ -4,4 +4,4 @@ metadata:
   name: custom-secret-scripts
 type: Opaque
 data:
-  timescale-secrets.sh: IyEvYmluL2Jhc2gKcHNxbCAtZCAiJDEiIC0tZmlsZT0tIC0tc2V0IE9OX0VSUk9SX1NUT1A9MSA8PCBfX1NRTF9fClNFVCBsb2dfc3RhdGVtZW50IFRPIG5vbmU7ICAgICAgLS0gcHJldmVudCB0aGVzZSBwYXNzd29yZHMgZnJvbSBiZWluZyBsb2dnZWQKQUxURVIgVVNFUiBoYWNrYXRob24gIFdJVEggUEFTU1dPUkQgJ2hhY2thdGhvbic7Cl9fU1FMX18=
+  timescale-secrets.sh: IyEvYmluL2Jhc2gKcHNxbCAtZCAiJDEiIC0tZmlsZT0tIC0tc2V0IE9OX0VSUk9SX1NUT1A9MSA8PCBfX1NRTF9fClNFVCBsb2dfc3RhdGVtZW50IFRPIG5vbmU7ICAgICAgLS0gcHJldmVudCB0aGVzZSBwYXNzd29yZHMgZnJvbSBiZWluZyBsb2dnZWQKQUxURVIgVVNFUiBoYWNrYXRob24gIFdJVEggUEFTU1dPUkQgJ2hhY2thdGhvbic7CkFMVEVSIFVTRVIgY2l0eWRhc2hib2FyZCAgV0lUSCBQQVNTV09SRCAnY2l0eWRhc2hib2FyZCc7Cl9fU1FMX18K
```

Comment:
We should in future create a better alternative than just hardcoding the password there

Reply:
How do I update this value? A short documentation would be nice!